maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderController.java

```diff
@@ -90,6 +97,17 @@ public ResponseEntity<RepOrderDTO> updateApplicationDateCompleted(@RequestBody U
         return ResponseEntity.ok(repOrderService.updateDateCompleted(updateAppDateCompleted));
     }
 
+    @GetMapping(value = "/rep-order-usn/{usn}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
@lkm0287 as the resource being accessed is still a `rep-order` could we not just use a `@RequestParam` so the endpoint would be `/rep-orders?usn=x` instead of `/rep-order/rep-orders-usn/x`?

Reply:
Thanks, agree :) updated.